Samples/2a-arm-validation/readme.md

```diff
@@ -14,14 +14,17 @@ input-file:
 This time, we not only want to generate code, we also want to validate.
 
 ``` yaml
-azure-arm: true # enables validation messages
+azure-validator: true # Azure specific validation
+model-validator: true # validation of examples against the API definition
+semantic-validator: true # general semantic checks such as: Does the specified `default` value actually have the type specified for the corresponding field?
 ```
 
 ## Generation
 
 Also generate for some languages.
 
 ``` yaml 
+azure-arm: true # generate code using Azure ARM flavor
 csharp:
   output-folder: CSharp

```

Comment:
oh dear... do you know how that can easily be done? the thing is that my git is configured to not change line endings which is usually what you want except right now :D will see